frontend/.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
This might work with vscode;s json parser. But there is no such things as comments in json

Reply:
Those comments were automatically generated by vscode when it created the file. Vscode can parse them and _only_ vscode should try to use this file. It is in the .vscode folder for that reason.